inventory/any.cf

```diff
@@ -520,38 +520,30 @@ bundle agent cfe_autorun_inventory_aws_ec2_metadata
 {
   methods:
     !(disable_inventory_aws|disable_inventory_aws_ec2_metadata)::
-      "cfe_autorun_inventory_aws_ec2_metadata_cache";
+      "cfe_autorun_inventory_aws_ec2_metadata_cache"
+        action => if_elapsed_day;
       "cfe_aws_ec2_metadata_from_cache";
 }
 bundle agent cfe_autorun_inventory_aws_ec2_metadata_cache
-# @brief Cache ec2 metadata from http request for one day
+# @brief Cache ec2 metadata from http request
+# It's up to caller to ensure this bundle is called only once a day
```

Comment:
Note: Only calling this bundle once a day will not prevent url_get from running each time the agent is executed.

Reply:
Sorry, not line 582, I meant to say line 528. DYSLEXIA!